src/styles/SearchResults.css

```
@@ -0,0 +1,22 @@
+.image-gallery {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap ;
+    justify-content: center;
+    column-gap: 20px;
+    row-gap: 20px;
+    margin: 2em 20em 20em 20em;
```

Minor point but this could be put into a shorthand form:
```suggestion
    margin: 2em 20em 20em;
```